### ISSUE TITLE
Providing email address to enter polite pool

### ISSUE THREAD

**Issue by nealhaddaway** [KTH-Library/openalex#1]
Amazing package! Really great. I was trying to work out how to provide an email address to enter the polite pool - can you do that within the openalex_crawl() function? Thanks!

**Comment by mskyttner** [comment]
Hi! This package is kind of experimental, great if it can be useful...

For entering the polite pool, the email can be provided by setting the OPENALEX_USERAGENT environment var to include the email like "mailto:you@example.com". According to https://docs.openalex.org/api#the-polite-pool. 

I added a helper fcn, see https://github.com/KTH-Library/openalex#rate-limits, which can set an unset an email to use for the session and which informs about how to use .Renviron to make the setting sticky.

In some quick tests it looks like it can be slightly faster to provide the email. My first runs with some random works were inconclusive but with some crawls I saw a difference like 25% time saved if being in the polite pool versus common pool.
